webapp/packages/core-blocks/src/Containers/GroupTitle.tsx

```diff
@@ -15,16 +18,32 @@ import elementsSizeStyles from './shared/ElementsSize.module.css';
 interface Props {
   sticky?: boolean;
   header?: boolean;
+  onClose?: () => void;
 }
 
 export const GroupTitle: React.FC<Props & ILayoutSizeProps & React.HTMLAttributes<HTMLHeadingElement>> = function GroupTitle({
   sticky,
   header,
   className,
+  children,
+  onClose,
   ...rest
 }) {
+  const translate = useTranslate();
   const style = useS(styles, elementsSizeStyles);
   const divProps = filterLayoutFakeProps(rest);
   const layoutProps = getLayoutProps(rest);
-  return <h2 tabIndex={-1} {...divProps} className={s(style, { groupTitle: true, sticky, header, ...layoutProps }, className)} />;
+
+  return (
+    <h2 tabIndex={-1} {...divProps} className={s(style, { groupTitle: true, sticky, header, ...layoutProps }, className)}>
+      {onClose ? (
+        <Flex gap="xs" align="center">
+          <ActionIconButton className={s(styles, { closeButton: true })} name="angle" title={translate('ui_close')} onClick={onClose} />
+          {children}
+        </Flex>
+      ) : (
+        children
+      )}
+    </h2>
+  );
```

Comment:
please remove, use composition instead, another question we have `GroupClose` component what is wrong with it?